assets/src/components/activities/common/authoring/actions/stemActions.ts

```diff
@@ -5,6 +5,13 @@ import { PREVIEW_TEXT_PATH, STEM_PATH } from 'data/activities/model/utils';
 import { Operations } from 'utils/pathOperations';
 
 export const StemActions = {
+  changeEditorMode(mode: 'slate' | 'markdown') {
+    return (model: any) => {
+      model.stem.editor = mode;
```

Comment:
TODO: I need to go back and revisit this, it's not currently handling the stemPath right.